integration_tests.py

```diff
@@ -593,7 +593,6 @@ def _check_containers_running(*, is_up=True):
             ["docker", "compose", "-f", docker_compose_fn, "ps", "-q", "-a"],
```

Comment:
Can you try this instead?

```suggestion
            ["docker", "compose", "-f", str(docker_compose_fn), "ps", "-q", "-a"],
```

Reply:
I've tried but I get the same error.

Reply:
The error is not a python error.

docker compose ps just has a non-zero exit code when no containers are running